### ISSUE TITLE
Reward used in the paper

### ISSUE THREAD

**Issue by wzaielamri** [gauthamvasan/modulated_locomotor_controllers#1]
Hello @gauthamvasan ,
Did you succeed in reproducing the paper of Heess? 
Do you know what is the specific reward they used? (in the paper, it is written that they used the negative of the distance). However, is it the same reward for the high-level controller and also low-level control in the pre-training phase?

It would be really great if you can share with me your knowledge.
